deebot_client/commands/json/auto_empty.py

```diff
@@ -0,0 +1,41 @@
+"""Auto empty command module."""
+
+
+from typing import Any
+
+from deebot_client.command import InitParam
+from deebot_client.event_bus import EventBus
+from deebot_client.events.auto_empty import AutoEmptyMode, AutoEmptyModeEvent
+from deebot_client.message import HandlingResult, MessageBodyDataDict
+
+from .common import CommandWithMessageHandling, SetCommand
+
+
+class GetAutoEmpty(CommandWithMessageHandling, MessageBodyDataDict):
+    """Get auto empty command."""
+
+    name = "getAutoEmpty"
+
+    @classmethod
+    def _handle_body_data_dict(
+        cls, event_bus: EventBus, data: dict[str, Any]
+    ) -> HandlingResult:
+        """Handle message->body->data and notify the correct event subscribers.
+
+        :return: A message response
+        """
+        event_bus.notify(AutoEmptyModeEvent(AutoEmptyMode(str(data["frequency"]))))
```

Comment:
We should also return if auto empty is enabled.
For example in `GetWaterInfo` I use the event also for two entities in HA and I suggest we do here the same. So we can remove `deebot_client/commands/json/auto_empty_enable.py` completely

Reply:
Did i understand you correct, i make here a change:
https://github.com/MVladislav/client.py/commit/f4a4d4d22aff6ef1fce7507d48a5907f23e222b4